std/traits.d

```diff
@@ -5489,6 +5489,27 @@ enum bool isBoolean(T) = is(BooleanTypeOf!T) && !isAggregateType!T;
  */
 enum bool isIntegral(T) = is(IntegralTypeOf!T) && !isAggregateType!T;
 
+///
+@safe unittest
+{
+    static assert(isIntegral!byte);
+    static assert(isIntegral!short);
+    static assert(isIntegral!int);
+    static assert(isIntegral!long);
+
+    static assert(!isIntegral!double);
```

Comment:
For compactness you may group all true asserts together and all false asserts together:

```d
static assert(isIntegral!byte && isIntegral!short && ...);
static assert(!isIntegral!double && ...);
```

In fact if you exhaustively enumerate all cases for `true` there's no need for example for `false`.